Rakefile

```diff
@@ -5,9 +5,6 @@ require 'bundler/gem_tasks'
 
 Bundler::GemHelper.install_tasks
```

Comment:
Do we need this?

Reply:
No, it’s now called when you require `gem_tasks`: https://github.com/rubygems/rubygems/blob/bce0fb2fb0294c0a02d78ff28097695b20002e8c/bundler/lib/bundler/gem_tasks.rb#L7